src/me/mrCookieSlime/Slimefun/SlimefunGuide.java

```diff
@@ -562,7 +551,6 @@ else if (!(category instanceof LockedCategory)) {
 						if (!(category instanceof SeasonCategory)) {
 							menu.addItem(index, category.getItem());
 							menu.addMenuClickHandler(index, new MenuClickHandler() {
```

Comment:
I suspect the file size could be reduced even further by replacing all MenuClickHandlers with
(p, slot, item, action) -> false

Reply:
Alright, I'll do this.
Also, I can make a separate PR later using diamond operators everywhere if you don't mind.